libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -36,21 +52,65 @@ export function toFragment(htmlStrings, ...values) {
   return fragment;
 }
 
-export const getFedsPlaceholderConfig = () => {
-  const { locale } = getConfig();
-  let libOrigin = 'https://milo.adobe.com';
+let federatedContentRoot;
+export const getFederatedContentRoot = () => {
+  if (federatedContentRoot) return federatedContentRoot;
+
   const { origin } = window.location;
+  const allowedOrigins = [
+    'https://www.adobe.com',
+    'https://business.adobe.com',
+    'https://blog.adobe.com',
+    'https://milo.adobe.com',
+  ];
+
+  federatedContentRoot = allowedOrigins.some((o) => origin === o)
+    ? origin
+    : 'https://www.adobe.com';
```

Comment:
@overmyheadandbody @narcis-radu do you see any issues with this approach of falling back to www to retrieve federal content?

This way if there are any consumers we are unaware of - they can continue to function and also consumers newly onboarding to milo don't need to think about adding CDN proxies to start out.

Reply:
I don't see any issues, I consider this to be better than `milo.adobe.com`.